staking_deposit/intl/en/cli/generate_bls_to_execution_change.json

```diff
@@ -0,0 +1,41 @@
+{
+    "generate_bls_to_execution_change": {
+        "arg_generate_bls_to_execution_change" :{
+            "help": "Generating the SignedBLSToExecutionChange data to enable withdrawals on Ethereum Beacon Chain."
+        },
+        "arg_execution_address": {
+            "help": "The 20-byte (Eth1) execution address that will be used in withdrawal",
+            "prompt": "Please enter the 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain.",
+            "confirm": "Repeat your execution address for confirmation.",
+            "mismatch": "Error: the two entered values do not match. Please type again."
+        },
+        "arg_validator_indices": {
+            "help": "A list of the validator index number(s) of the certain validator(s)",
+            "prompt": "Please enter a list of the validator index number(s) of your validator(s) as identified on the beacon chain. Split multiple items with whitespaces or commas."
+        },
+        "arg_bls_withdrawal_credentials_list": {
```

Comment:
Providing anything other than hex for this argument will throw an unhanded exception:
```
user@laptop:~/Downloads/staking_deposit-cli-d83c312-linux-amd64$ ./deposit generate-bls-to-execution-change 

***Using the tool on an offline and secure device is highly recommended to keep your mnemonic safe.***

Please choose your language ['1. العربية', '2. ελληνικά', '3. English', '4. Français', '5. Bahasa melayu', '6. Italiano', '7. 日本語', '8. 한국어', '9. Português do Brasil', '10. român', '11. Türkçe', '12. 简体中文']:  [English]: 3
Please choose the (mainnet or testnet) network/chain name ['mainnet', 'goerli', 'sepolia', 'zhejiang']:  [mainnet]: 
Please enter your mnemonic separated by spaces (" "). Note: you only need to enter the first 4 letters of each word if you'd prefer.: 
Please enter your mnemonic separated by spaces (" "). Note: you only need to enter the first 4 letters of each word if you'd prefer.: never ski use pet common nose pipe assault cloth renew cement save avoid wing attract news bid priority dash replace shield pair candy smile
Please enter the index position for the keys to start generating withdrawal credentials in ERC-2334 format. [0]: 0
Please enter a list of the validator index number(s) of your validator(s) as identified on the beacon chain. Split multiple items with whitespaces or commas.: 1
Please enter a list of the old BLS withdrawal credentials of your validator(s). Split multiple items with whitespaces or commas.: 1
Traceback (most recent call last):
  File "staking_deposit/deposit.py", line 63, in <module>
  File "click/core.py", line 829, in __call__
  File "click/core.py", line 782, in main
  File "click/core.py", line 1257, in invoke
  File "click/core.py", line 700, in make_context
  File "click/core.py", line 1048, in parse_args
  File "click/core.py", line 1630, in handle_parse_result
  File "click/core.py", line 123, in invoke_param_callback
  File "staking_deposit/utils/click.py", line 99, in callback
  File "staking_deposit/cli/generate_bls_to_execution_change.py", line 104, in <lambda>
  File "staking_deposit/utils/validation.py", line 246, in validate_bls_withdrawal_credentials_list
  File "staking_deposit/utils/validation.py", line 246, in <listcomp>
  File "staking_deposit/utils/validation.py", line 226, in validate_bls_withdrawal_credentials
  File "staking_deposit/utils/validation.py", line 213, in normalize_bls_withdrawal_credentials_to_bytes
ValueError: non-hexadecimal number found in fromhex() arg at position 1
[20313] Failed to execute script 'deposit' due to unhandled exception!
```